crypto/decrepit/cfb/cfb.c

```diff
@@ -36,6 +40,55 @@ static int aes_cfb_init_key(EVP_CIPHER_CTX *ctx, const uint8_t *key,
   return 1;
 }
 
+static int aes_cfb1_cipher(EVP_CIPHER_CTX *ctx, uint8_t *out,
+                            const uint8_t *in, size_t len) {
+  if (!out || !in) {
+    return 0;
+  }
+
+  EVP_CFB_CTX *cfb_ctx = ctx->cipher_data;
```

Comment:
Should cast this since `ctx->cipher_data` is a void pointer.

Reply:
Will consider. I didn't see compiler complains that.

Reply:
Added explicit cast.